core/runtime/c/cgo_scontainer.c

```diff
@@ -61,33 +61,39 @@ __attribute__((constructor)) static void init(void) {
     int ret;
 
     if ( stage <= 0 ) {
-        pfatal("STAGE environement variable not set");
+        singularity_message(ERROR, "STAGE environement variable not set\n");
```

Comment:
why not use one function like pfatal or something like `singularity_abort` rather than always call exit() just after ?